src/server/routes/__tests__/catches.integration.spec.js

```diff
@@ -137,6 +138,29 @@ describe('catches.integration', () => {
       expect(createdCatch.statusCode).toBe(400)
     })
 
+    it('should validate succesfully if the "method" is internal and the user is an admin or fmt', async () => {
```

Comment:
As with https://github.com/DEFRA/rod-catch-returns-api-js/pull/94/, do we need a test against the FMT role as well?